en/micro-integrator/docs/references/connectors/as400-pcml-connector/as400-pcml-connector-configuration.md

```diff
@@ -0,0 +1,65 @@
+# Setting up the AS400 PCML Environment  
+
+The AS400 PCML connector allows you to access RPG programs that are available on AS400 (renamed as IBM iSeries) servers using WSO2 EI. This is done using [Program Call Markup Language](https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_74/rzahh/pcml.htm) (PCML). 
+
+To work with the AS400 PCML connector, you need to have a IBM iSeries server. If you do not have a IBM iSeries server, go to [public IBM iSeries server](https://pub400.com/), and create a public IBM iSeries server.
+
+The connector uses the IBM JTOpen library for all its operations. Copy the **jt400.jar** to `<ESB_HOME>/repository/components/lib` folder. You can download the IBM JTOpen library from (here)[https://sourceforge.net/projects/jt400/].
+
+### Setting up public IBM iSeries server
+
+Follow the steps below to create public IBM iSeries server.
+
+1. Go to [https://pub400.com](https://pub400.com/) and click **Sign up** button.
+
+    <img src="../../../../assets/img/connectors/as400-signup.png" title="AS400-signup page" width="600" alt="AS400-signup page"/>
+    
+2. Enter required details for request a new user profile and click **Submit** button.
+
+    <img src="../../../../assets/img/connectors/as400-signup-details.png" title="AS400-signup details" width="600" alt="AS400-signup details"/>
+    
+3. You will received the reply email with the client credentials to the your email account (It will take some hours).
+
+4. Navigate in to the [IBM i Access - Client Solutions](https://www.ibm.com/support/pages/ibm-i-access-client-solutions) page.
+
+5. Download the **IBM i Access Client Solutions**.
+
+    <img src="../../../../assets/img/connectors/as400-ibm-client-access.png" title="IBM i Access Client Solutions" width="600" alt="IBM i Access Client Solutions"/>
+
+6. Extract the downloaded zip file and install it.
+
+7. Start the IBM I access client solution.
+   
+    <img src="../../../../assets/img/connectors/as400-ibm-client-access-homepage.png" title="IBM i Access Client Solutions start" width="600" alt="IBM i Access Client Solutions start"/>
+
+8. Click **System Configurations** section under **Management** section. 
+
+    <img src="../../../../assets/img/connectors/as400-system-configuration.png" title="IBM i Access system-configuration" width="500" alt="IBM i Access system-configuration"/>
+
+9. You will prompt following window and click **New** button.
```

Comment:
```suggestion
9. The following window appears. Click the **New** button.
```

Reply:
Ideally we need to add more information on what we are trying to achieve in this step.

Reply:
Updated.